waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/DatabaseMapping.java

```diff
@@ -147,6 +148,8 @@
 
   GetTablesResult transformOutboundGetTablesResult(GetTablesResult result);
 
+  PartitionValuesRequest transformInboundPartitionValuesRequest(PartitionValuesRequest req);
```

Comment:
Shouldn't there be an accompanying transformOutboundPartitionValuesRequest method?

Reply:
that would be a outbound response and no looks like the response doesn't have a db name field

Reply:
Ah, OK, yes, that makes sense, so we don't need any of the outbound changes then.